rosdep/python.yaml

```diff
@@ -1797,6 +1797,52 @@ python-hypothesis:
     yakkety_python3: [python3-hypothesis]
     zesty: [python-hypothesis]
     zesty_python3: [python3-hypothesis]
+python-imageio:
+  alpine:
+    pip:
+      packages: [imageio]
+  arch:
+    pip:
+      packages: [imageio]
+  cygwin:
+    pip:
+      packages: [imageio]
+  debian:
+    '*':
+      pip:
+        packages: [imageio]
+    buster: [python-imageio]
+  fedora:
+    pip:
+      packages: [imageio]
+  freebsd:
+    pip:
+      packages: [imageio]
+  gentoo: [dev-python/imageio]
+  macports:
+    pip:
+      packages: [imageio]
+  opensuse:
+    '*':
+      pip:
+        packages: [imageio]
+    '15.0': [python-imageio]
+  osx:
+    pip:
+      packages: [imageio]
+  rhel:
+    pip:
+      packages: [imageio]
+  slackware:
+    pip:
+      packages: [imageio]
+  ubuntu:
+    '*':
```

Comment:
We definitely don't want a wildcard rule with the pip entry. The logic should be reversed and put the pip entries only on the older platforms.